### ISSUE TITLE
navigation and branding landmarks must have a unique role or role/label/title combination

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#880]
In testing, masthead / navbar landmarks on all pages (home page, about page, help page, contact page, collection level pages, browse collection pages, and catalog view) failed tests due to not having a unique role or role/label/title combination. 

[Deque aXe help page for this issue](https://dequeuniversity.com/rules/axe/4.3/landmark-unique). 
w3c [has](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html) some [guidance](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/banner.html) on this topic as well. 

Associated element / source code:
`<nav id="masthead" class="navbar navbar-inverse navbar-static-top" role="navigation">`

Associated selector: 
`["#masthead"]`

**Comment by noraegloff** [comment]
Deque aXe tools severity level = moderate
